cmd/goacme/main_test.go

```diff
@@ -22,15 +22,15 @@ func TestDiscoAliasFlag(t *testing.T) {
 		{defaultDisco, []string{"-d", "https://disco"}, "https://disco"},
 	}
 	for i, test := range tests {
-		var a discoAliasFlag = test.a
+		var daf = test.a
```

Comment:
I can, any reason though? Just want to understand the reasoning. Go from how I've understood things usually follows the principle of using the first letter from each major word in the represented type / variable. In this case `discoAliasFlag` becomes `daf`.